src/ghostwriter/hooks/github_notebook.py

```diff
@@ -29,32 +26,42 @@ async def github_notebook(params: Parameters) -> Parameters | None:
         # We know the stream output should be the serial number and
         # a newline.
         serial = (await lab_session.run_python(code)).strip()
-    if serial and serial != "0" and params.target is not None:
-        #
-        # params.target is never None, because it's injected by run_hooks(),
-        # but mypy doesn't know that.
-        #
-        # We had to change the filename and filename schema.
-        # Return a modified Parameters object.
-        #
-        targetcomp = params.target.split("/")
-        targetcomp[-1] = "${path}-${unique_id}.ipynb"
-        target = "/".join(targetcomp)
-        unique_id = str(serial)
-        LOGGER.debug(
-            "Continuing to redirect", target=target, unique_id=unique_id
-        )
-        return Parameters(
-            user=params.user,
-            base_url=params.base_url,
-            token=params.token,
-            client=params.client,
-            path=params.path,
-            target=target,
-            unique_id=unique_id,
-        )
-    LOGGER.debug("Continuing to redirect; params unchanged")
-    return None
+
+    # Honestly it's easier to just unconditionally rewrite the target
+    # than to figure out whether it needs rewriting.
+
+    # Start with the common fragment
+    target = (
```

Comment:
No, because we're using it purely as a string for constructing a URL to return.  At least, that's what I think.  All the parts where it would be a Path are in the underscore-named payload it runs in the lab.